content/concepts/transport.md

```diff
@@ -115,3 +115,189 @@ and some places in the codebase still use the "swarm" terminology.
+```
+
+In this section, we offered an overview of QUIC and how QUIC works in libp2p.
+
+{{% notice "tip" %}}
+
+For more details on QUIC, including its limitations 
+check out the following resources:
+
+{{% /notice %}}
+
+## WebTransport
+
+Another transport protocol under development at the IETF is WebTransport.
+WebTransport is a new specification that uses QUIC to offer an alternative to
+WebSockets. Instead, it can be considered WebSockets over QUIC by allowing 
+browsers to establish a stream-multiplexed and bidirectional 
+connection to servers. 
+
+The specification can depend on and reuse the QUIC infrastructure in place 
+to offer WebSockets that take the benefits of UDP and offer sockets without head-of-line 
+blocking.
+
+Recall that WebSockets are bidirectional, full-duplex communication between two 
+points over a single-socket connection. WebTransports, as a result, can be used
+like WebSockets, but with the support of multistreams.
+
+WebTransport streams can be arbitrary in size and independent when possible. 
+They are reliable but can be canceled when possible. The datagrams in a 
+WebTransport connections are MTU-sized and can be unreliable when possible.
+
+{{% notice "caution" %}}
+
+There is a functioning WebTransport implementation in go-libp2p that is part 
+of the v0.23 release.
+
+The implementation should be used experimentally and is not recommended for
+production environments.
+
+{{% /notice %}}
+
+For network stacks like libp2p, WebTransport is a pluggable
+protocol that fits well with a modular network design.
+
+For a standard WebSocket connection:
+
+- 1 RTT for DNS resolution
+- 1 RTT for TCP handshake
+- 1 RTT for TLS handshake
+- 1 RTT for WebSocket handshake
+- 1 RTT for Multistream Security handshake
+- 1 RTT for libp2p handshake
+
+Plenty of handshakes and roundtrips: 6 RTTs: 5 handshakes + 1 DNS resolution
+
+WebTransport running over QUIC only requires 4 RTTs, as:
+
+- 1 RTT for QUIC handshake
+- 1 RTT for WebTransport handshake
+- 2 RTT for libp2p handshake; one for multistream and one for the secure 
+  communication (TLS 1.3 or Noise)
+
+> With protocol select, the WebTransport handshake and the libp2p handshake 
+> can run in parallel, bringing down the total round trips to 2.
+
+WebTransport multiaddresses are composed of a QUIC multiaddr, followed 
+by `/webtransport` and a list of multihashes of the node certificates that the server uses.
+
+For instance, for multiaddress `/ip4/127.0.0.1/udp/123/quic/webtransport/certhash/<hash1>`, 
+a standard local QUIC connection is defined up until and including `/quic.` 
+Then, `/webtransport/` runs over QUIC and the self-signed certificate hash that the 
+server will use to verify the connection.
+
+WebTransport requires an HTTPS URL to establish a WebTransport session - 
+e.g., `https://docs.libp2p.com/webtransport` and the multiaddresses use an HTTP URL
+instead. Since multiaddrs don't allow the encoding of URLs, the HTTP endpoint of a libp2p 
```

Comment:
> Since multiaddrs don't allow the encoding of URLs,

This isn't exactly true. We just haven't defined what it would look like. Multiaddrs are able to encode urls, we just haven't come to consensus on what a canonical representation here looks like.

Reply:
Addressed